docs/styleguides/STYLEGUIDE.md

```
@@ -314,22 +336,42 @@ Use as little jargon as necessary. Describe jargon in parentheses on first refer
 
 > ✅ In your browser, open [`http://localhost:3000/`](http://localhost:3000/).
 
-### Specify syntax and filename for terminal commands
+### Specify syntax for terminal commands
 
-If the code should run in a terminal, set the code block's syntax highlighting and filename appropriately.
+If the code should run in a terminal, set the code block's syntax highlighting appropriately.
 
 > ❌
 
 ````
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)